tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -42,18 +85,22 @@ For each endpoint, you can write SQL statements to execute on a TiDB cluster, de
 
 On the right pane of the endpoint details page, you can click the **Properties** tab to view and manage the following properties of the endpoint:
 
-- **Endpoint Path**: the unique path of the endpoint that users use to access it.
+- **Path**: the path of the endpoint that users use to access the endpoint.
 
-    - The path must be unique within a Data App.
+    - The combination of the request method and the path must be unique within a Data App.
     - Only letters, numbers, underscores (`_`), and slashes (`/`) are allowed in the path, which must start with a slash (`/`) and end with a letter, number, or underscore (`_`). For example, `/my_endpoint/get_id`.
```

Comment:
Only letters, numbers, underscores (`_`), and slashes (`/`) are allowed in a path. A path must start with a slash (`/`) and end with a letter, number, or underscore (`_`). For example, `/my_endpoint/get_id`.